website/config.js

```diff
@@ -69,12 +69,9 @@ const config = {
       // { name: 'Prisma Enterprise', link: '/' },
     ],
     community: [
-      { name: 'Meet the community', link: 'https://www.prisma.io/community' },
-      { name: 'Slack', link: 'https://slack.prisma.io/' },
-      { name: 'Github', link: 'https://github.com/prisma' },
-      { name: 'Discussions', link: 'https://github.com/prisma/prisma/discussions' },
+      { name: 'GitHub', link: 'https://github.com/graphql-nexus' },
+      { name: 'GitHub Discussions', link: 'https://github.com/graphql-nexus/nexus/discussions' },
       { name: 'GraphQL Meetup', link: 'https://www.meetup.com/graphql-berlin/' },
-      { name: 'TypeScript Meetup', link: 'https://www.meetup.com/TypeScript-Berlin/' },
     ],
     resources: [
```

Comment:
These still need to be updated

Reply:
I'm only displaying the GraphQL meetup link:

![image](https://user-images.githubusercontent.com/849648/85673078-cf3d4b00-b6c3-11ea-9b80-842808839436.png)

But I will remove from config too.

Reply:
I think this comment was about the `resources` block.

Reply:
Oh! My mistake.